examples/databricks/example_databricks_workflow.py

```diff
@@ -53,19 +53,23 @@
         databricks_conn_id=DATABRICKS_CONN_ID,
         job_clusters=job_cluster_spec,
         notebook_params=[],
+        notebook_packages=[
+            {"pypi": {"package": "simplejson"}},
+        ],
     )
     with task_group:
         notebook_1 = DatabricksNotebookOperator(
             task_id="notebook_1",
             databricks_conn_id=DATABRICKS_CONN_ID,
-            notebook_path="/Users/daniel@astronomer.io/Test workflow",
+            notebook_path="/Shared/Notebook_1",
+            notebook_packages=[{"pypi": {"package": "Faker"}}],
```

Comment:
What do you think if we have one of the examples with the PyPI package version and perhaps the URL?

Reply:
added the example for simplejson at the taskgroup level